pkg/kubelet/metrics/metrics.go

```diff
@@ -190,7 +192,7 @@ var (
 			Buckets:        podStartupDurationBuckets,
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{},
+		[]string{"first_with_network"},
```

Comment:
I am not a fan of adding a label to the pod startup latency metric since it makes it harder to understand compare to having a standalone metric. It removes the value of having a dedicated documentation explaining how to use the metric. And although it happens to that network latency needs to be measured at the pod startup level, it is a different information so it should be in its own metric.

Reply:
I disagree, if you're unaware of this dimension, the lyrics will just 'work as intended'.

Reply:
I don't see the point of having that info here, it is an implementation detail that can only be understood by developers.
Also, I think that we always want to treat the first pod like all the others since having it as part of the overall latency can help surfacing and catching network issues via the metric that Antonio is adding in https://github.com/kubernetes/kubernetes/pull/121720